ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/vm/operations/SStoreOperation.java

```diff
@@ -27,8 +27,14 @@
 
 public class SStoreOperation extends AbstractOperation {
 
-  public SStoreOperation(final GasCalculator gasCalculator) {
+  public static final Gas FRONTIER_MINIMUM = Gas.ZERO;
+  public static final Gas EIP_1706_MINIMUM = Gas.of(2300);
```

Comment:
If we follow the existing pattern, I think these gas values belong on the `GasCalculator`?

Reply:
see https://github.com/PegaSysEng/pantheon/pull/1704#discussion_r303614431